topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,363 @@
+
+In this tutorial, we will use the maximum likelihood method [RAxML](https://cme.h-its.org/exelixis/web/software/raxml/) to estimate a phylogenetic tree for the 20 strains.
+
+An aspect we ignore in this tutorial is the uncertainty involved in phylogenetic inference. While RAxML will deliver a single tree, not all aspects of this tree are equally well supported by the data. This uncertainty can be quantified through **bootstrapping**, a procedure where are large number of trees are estimated from random samples of the original data. If a certain split in the original tree is present in all the bootstrapped trees, then we can be confident about this split. Published phylogenies should always include a measure of uncertainty, while for this tutorial you will have to believe me that we are looking at a solid phylogeny...
+
+## Hands-on
+> ### {% icon hands_on %} Hands-on: Estimate a phylogeny for 20 MTBC strains
+>
+> 1. {% tool [Phyogenetic reconstruction with RAxML](toolshed.g2.bx.psu.edu/repos/iuc/raxml/raxml/8.2.4+galaxy2) %} with the following parameters:
+>    - {% icon param-file %} *"Source file with aligned sequences"*: `output` (Input dataset)
+>    - *"Model type"*: `Nucleotide`
+>    - *"RAxML options to use"*: `Required options only`
+>
+> 2. The RAxML output we are interested in is the "Best-scoring ML tree". Select it in you Galaxy history and take a look at it with the different visualization options offered by Galaxy.
+>
+{: .hands_on}
+
+
+# Visualize and manipulate the tree
+Phylogenetic trees are great tools because they are at the same time quantitative (we can do calculations on branch lengths, estimate uncertainty of a tree topology etc.) and visually appealing, allowing to actually "see" biologically interesting patterns. Often this requires some tweaking of the tree, for example by coloring parts of the tree according to some background information we have about the samples. Here, to obtain an overview of the different strains present in the country (given the limitations of the sampling), our goal is to create a rooted tree in which colors indicate different MTB lineages.
+
+We will use R to plot and manipulate the phylogeny obtained from RAxML. The code to produce the figures is shown in the boxes below. You can execute it by starting RStudio within Galaxy, as explained [here]({% link topics/galaxy-interface/tutorials/rstudio/tutorial.md %}). This is not required to finish this tutorial, but if you have used R before, it might be worthwile to go through the code, modify it, and explore the numerous phylogenetics packages and functions in R.
```

Comment:
```suggestion
We will use R to plot and manipulate the phylogeny obtained from RAxML. The code to produce the figures is shown in the boxes below. You can execute it by starting RStudio within Galaxy. This is not required to finish this tutorial, but if you have used R before, it might be worthwile to go through the code, modify it, and explore the numerous phylogenetics packages and functions in R.

> ### {% icon hands_on %} Hands-on: Starting RStudio
>
> 1. Note the history IDs of the "Best-scoring ML tree"
>
> 2. {% tool [RStudio](interactive_tool_rstudio) %} in Galaxy provides some special functions such as `gx_get` to import and export files from your history.
>
>    {% snippet faqs/galaxy/interactive_tools_rstudio_launch.md %}
>
> 3. Copy the files we need into our workspace so we can see them in the Files pane.
>
>    ```R
>    treefile <- gx_get(1)  # Use the correct ID for your differentially expressed results dataset.
>    ```
>
{: .hands_on}
```

Reply:
Sorry @cstritt when I suggested using `{% link` I wasn't reading closely enough, here's a more GTN-standard way to tell users to launch RStudio

Reply:
This looks good, thanks!